docs/superpowers/plans/2026-05-05-chat-runtime-server-replay.md

```
@@ -0,0 +1,2905 @@
+
+The response is:
+
+```json
+{
+  "ok": true,
+  "sessionKey": "agent:main:main",
+  "runId": "run-id-from-openclaw",
+  "cursor": "runtime-cursor"
+}
+```
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)